### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://nikushaSheremet.github.io/layout_calendar/)

### CODE REVIEW

**Review by wfurs0w — APPROVED** [review]
Excellent work) I have no comments

**Comment by ghost** [comment]
thank you for your approve)

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar-wrapper">
+      <div class="container">
+        <div class="calendar calendar--month-length--31">
+          <div class="calendar__day calendar__day--start-day--sat"></div>
```
Comment (AlinaMatuschak) [inline src/index.html]: check task description: start-day should be a modifier for calendar (not for calendar__day)

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @include square($day-width);
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: transform 0.5s;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/calendar.scss]: Maybe background color should change smoothly too

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Good work 👏. Just add start-day modifier to calendar block

Should display a calendar in the middle of the screen (vertical too)
Screenshot 2022-07-22 at 12 23 24

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -24,14 +30,16 @@
 
     font-family: Arial, Helvetica, sans-serif;
     font-size: 30px;
-    transition: transform 0.5s;
 
     border: 1px solid #000;
     background: #eee;
 
+    transition-duration: 0.5s;
```
Comment (AndriiRut) [inline src/styles/blocks/calendar.scss:37]: look at your Demo. Now everything that happens with days block has a duration of the animation.
You shouldn't delete transform, but add color and etc.)

**Review by AndriiRut — APPROVED** [review]
Good job!

**Pull request closed without merging**